gui/tasks/forms.py

```diff
@@ -265,7 +300,7 @@ def clean(self):
         if 'rsync_user' in cdata and not (
             self.cleaned_data.get("rsync_mode") == 'module' or
             not self.cleaned_data.get("rsync_validate_rpath") or
-            self.check_rpath_exists()
+            self.check_rpath_exists() or self.ssh_host_keyscan()
```

Comment:
Again, this cannot be either/or and instead the user needs to hit the `SSH Keyscan` button above and then save the form at which point if  `Validate Remote Path:` is ticked then this just the `self.check_rpath_exists` will work just fine